Jenkinsfile

```diff
@@ -17,6 +59,21 @@ pipeline {
         junit 'test/junit.xml'
       }}
     }
+
+    stage('Release') {
+      when {
+        expression {
+          MODE == "RELEASE"
+        }
+      }
+
+      steps {
+        release { billOfMaterialsDirectory, assetDirectory ->
+          sh './publish-rubygem.sh'
```

Comment:
While not required, it's probably good practice to copy the resulting gem file to the assetDirectory.  See https://github.com/cyberark/conjur-api-ruby/blob/main/Jenkinsfile#L157 for an example.

Reply:
To clarify a little bit, any files added to `assetDirectory` will get automatically attached to the github release, so by adding the gem file to the directory it'll be available directly from the GH release rather than just from rubygems